doc/interop-test-descriptions.md

```diff
@@ -517,6 +517,34 @@ Procedure:
 Asserts:
 * Call completed with status CANCELLED
 
+### timeout_on_sleeping_server
+
+This test verifies that an RPC request whose lifetime exceeds its configured timeout value will end with the DeadlineExceeded status.
+
+Server features:
+* [FullDuplexCall][]
+* [Compressable Payload][]
+* [Observe ResponseParameters.interval_us][]
+
+Procedure:
+ 1. Client calls FullDuplexCall with the following request and sets its timeout to 1ms.
+
+    ```
+    {
+      response_type: COMPRESSABLE
+      response_parameters:{
+        interval_us: 3000
+      }
+      payload:{
+        body: 27182 bytes of zeros
+      }
+    }
+    ```
+2. Server will sleep for the configured interval_us and client waits for the status.
+
+Asserts:
+* Call completed with status DeadlineExceeded.
```

Comment:
Let's call it DEADLINE_EXCEEDED, since that is the formatting in most languages (and in the proto that everyone copied from). Not a big deal though.